src/platform/EFR32/ConfigurationManagerImpl.cpp

```diff
@@ -64,12 +64,29 @@ CHIP_ERROR ConfigurationManagerImpl::Init()
     rebootCause = RMU_ResetCauseGet();
     RMU_ResetCauseClear();
 
-    // If the fail-safe was armed when the device last shutdown, initiate a factory reset.
+    // If the fail-safe was armed when the device last shutdown, initiate cleanup to the pending Fail Safe Context with
+    // which the fail-safe timer has been armed.
```

Comment:
Similar comments as in the Ameba implementation.... and it looks like we need to move this out of platform-specific code into shared code, no?